src/fn_call.rs

```diff
@@ -7,7 +7,8 @@ use syntax::attr;
 use crate::*;
 
 impl<'a, 'mir, 'tcx> EvalContextExt<'a, 'mir, 'tcx> for crate::MiriEvalContext<'a, 'mir, 'tcx> {}
-pub trait EvalContextExt<'a, 'mir, 'tcx: 'a+'mir>: crate::MiriEvalContextExt<'a, 'mir, 'tcx> {
+
+pub trait EvalContextExt<'a, 'mir, 'tcx: 'a + 'mir>: crate::MiriEvalContextExt<'a, 'mir, 'tcx> {
```

Comment:
I had deliberately not added an empty line here because these two really just form one "thing": a way to extend `MiriEvalContext` with more methods.